cli/src/semdep/parsers/util.py

```diff
@@ -0,0 +1,214 @@
+from base64 import b16encode
```

Comment:
Same, would be nice to have a toplevel comment.
Maybe you can link to some doc about this parsy, anything you find useful
to learn how to use it, so the next person reading this code can ramp up more quickly.